9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Sure, but there is documentation in the readme, isn't that enough? If we have an empty file, the "Executing post-init ..." line will get to the container logs. Therefore, it will always get printed out, unless the user overrides the hooks.d directory with a directory that is missing post-init. This just seems strange and I'd prefer we only log that line if we truly do execute a valid post-init that actually should do something.

An example would be nice, but I think that an example should actually do something useful and should therefore not be included directly in the image.